### PR TITLE
Make image UBI based

### PR DESCRIPTION
**Pull request by mmorhun**
Switch the image base from Alpine to UBI

Signed-off-by: Mykola Morhun <mmorhun@redhat.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,15 +6,15 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-FROM alpine:3.11.3
+FROM registry.access.redhat.com/ubi8:8.1-397
```
Comment (nickboldt) [inline Dockerfile]: Recommend adding a comment line above this (line 8):

`# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8`

That way when it's time to update this image, you can just run https://github.com/redhat-developer/codeready-workspaces/blob/master/product/updateBaseImages.sh and it'll automatically find the latest AND push a commit for you too.

Comment (nickboldt) [reply Dockerfile]: If you wanted to do more work here you could use ubi8-minimal (smaller base image) and then use `microdnf` to install the stuff you actually need in the container, like `microdnf -y install curl openssl` and whatever else the entrypoint needs (`bash`?)

**Review by nickboldt — APPROVED** [review]
+1 but could be improved if you want.

**Comment by nickboldt** [comment]
+1 as is: do you want more reviews, or would you like me to merge? is there a PR check that should be run here?

**Comment by mmorhun** [comment]
@nickboldt I am testing it with ubi-minimal (which is still 158 Mb)

**Review by AndrienkoAleksandr — APPROVED** [review]

**Comment by benoitf** [comment]
why a switch and not another Dockerfile as usual ?

**Comment by nickboldt** [comment]
because why maintain both when you don't have to ? :D UBI first!

**Comment by benoitf** [comment]
Well, because on upstream we're picking up smallest image ? :)

**Pull request merged**